source/unified-test-format/unified-test-format.rst

```diff
@@ -3748,114 +3743,91 @@ better processes can be established for versioning other specs *and* collating
 spec changes developed in parallel or during the same release cycle.
 
 
-Change Log
-==========
+Changelog
+=========
 
+..
+  Please note schema version bumps in changelog entries where applicable.
```

Comment:
This seemed like the kosher way to add an RST comment: https://stackoverflow.com/a/4783854/162228